src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -5,19 +5,25 @@
 import net.minecraft.item.Item;
 import net.minecraftforge.fml.common.registry.IForgeRegistry;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Class to create the modBlocks.
  * References to the blocks can be made here
  * <p>
  * We disabled the following finals since we are neither able to mark the items as final, nor do we want to provide public accessors.
  */
 @SuppressWarnings({"squid:ClassVariableVisibilityCheck", "squid:S2444", "squid:S1444", "squid:S1820",})
+
 public final class ModBlocks
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 38 to the maximum authorized 30 or less. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)